fmriprep/workflows/confounds.py

```diff
@@ -7,21 +7,24 @@
 Calculates frame displacement, segment regressors, global regressor, dvars, aCompCor, tCompCor
 '''
 from __future__ import print_function, division, absolute_import, unicode_literals
-
 from niworkflows.nipype.interfaces import utility
+from niworkflows.nipype.interfaces import fsl
 from niworkflows.nipype.algorithms import confounds
 from niworkflows.nipype.pipeline import engine as pe
+from niworkflows.nipype.interfaces.fsl import ICA_AROMA as aroma
 from niworkflows.interfaces.masks import ACompCorRPT, TCompCorRPT
+from niworkflows.interfaces import segmentation as nws
+
 
 from niworkflows.nipype.interfaces.nilearn import SignalExtraction
 from fmriprep.interfaces.utils import prepare_roi_from_probtissue
 
 
-def init_discover_wf(bold_file_size_gb, name="discover_wf"):
+def init_discover_wf(bold_file_size_gb, name="discover_wf",use_aroma=False):
```

Comment:
`use_aroma` should not have a default set here. This might be the core of the error you are getting.